Web/starter_project_barbies/components/common/Pagination.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react"
+
+interface PaginationItemProps {
+	pageNumber: number
+	isCurrentPage: boolean,
+	onPageClick: React.Dispatch<React.SetStateAction<number>>
+}
+
+interface PaginationProps {
+	numberOfPages: number
+}
+
+const PaginationItem: React.FC<PaginationItemProps> = ({pageNumber, isCurrentPage, onPageClick}) => {
```

Comment:
I would recommend defining any additional (unexported) components at the bottom

when i open the file i would prefer to see the main component definition first and see the others afterwards